composer.json

```diff
@@ -12,20 +11,24 @@
         }
     ],
     "require": {
-        "php": ">=5.5.0"
+        "php": "^5.5 || ^7.0"
```

Comment:
Personally, I have mixed feelings about this. When PHP7 came out it was extremely difficult for my colleagues and I to test because our builds on PHP7 because so many upstream packages specified `php: ^5.5` constraints, although, in practice, virtually all of these packages were compatible with PHP7. Internally, I had to fork a lot of packages while we waited for maintainers to update the PHP constraints in their _composer.json_ files. Following that experience, my personal preference is now to specify `php: >=#` with the view that incompatibilities with future PHP versions will be resolved with patches. It's worth bearing in mind that PHP very rarely break BC in a significant way.